examples/integration_test.py

```diff
@@ -13,24 +13,26 @@
 from aptos_sdk.account_address import AccountAddress
 from aptos_sdk.aptos_cli_wrapper import AptosCLIWrapper, AptosInstance
 
-from .common import APTOS_CORE_PATH
-
 
 class Test(unittest.IsolatedAsyncioTestCase):
     _node: Optional[AptosInstance] = None
+    _aptos_core_path: str
```

Comment:
why this refactor?

Reply:
If we import `common.py` here, 

```python
os.environ["APTOS_FAUCET_URL"] = "http://127.0.0.1:8081"
os.environ["APTOS_NODE_URL"] = "http://127.0.0.1:8080/v1"
```
of `setUpClass` method in `integration_test.py` will not be applied in each test case.

So `make integration_test` is basically running tests on devnet right now.